rest/src/main/kotlin/builder/message/modify/MessageModifyBuilder.kt

```diff
@@ -101,3 +116,15 @@ public inline fun MessageModifyBuilder.actionRow(builder: ActionRowBuilder.() ->
         it.add(ActionRowBuilder().apply(builder))
     }
 }
+
+/**
+ * Sets/Unsets the [MessageFlags] for this message.
+ *
+ * **Only supports [MessageFlag.SuppressEmbeds]**
+ */
+public inline fun MessageCreateBuilder.messageFlags(builder: MessageFlags.Builder.() -> Unit) {
```

Comment:
```suggestion
public inline fun MessageModifyBuilder.messageFlags(builder: MessageFlags.Builder.() -> Unit) {
```
this will fail to compile because `flags` is not a `var` in `MessageModifyBuilder`